apps/web/src/routes/_content/resources/-components/cards/resource-card.tsx

```diff
@@ -1,33 +1,64 @@
+import Flag from '#src/molecules/Flag/index.tsx';
+
 interface ResourceCardProps {
   imageSrc?: string | null;
   name: string;
   author: string;
   year?: number | null;
+  language?: string;
+  level?: string;
 }
 
 export const ResourceCard = (props: ResourceCardProps) => {
   return (
-    <div className="md:relative group md:max-w-64 flex md:flex-col gap-4 md:gap-8 p-3 md:p-0 max-md:hover:bg-darkOrange-10 hover:shadow-sm-card border grow border-transparent hover:border-darkOrange-6 rounded-2xl transition-all overflow-hidden">
+    <div className="md:relative group min-w-[288px] sm:min-w-[400px] md:min-w-[250px] md:max-w-64 flex md:flex-col gap-4 md:gap-8 px-2 py-1.5 md:p-0 max-md:hover:bg-darkOrange-10 hover:shadow-sm-card border grow shrink-0 border-transparent hover:border-darkOrange-6 rounded-2xl transition-all overflow-hidden">
       <img
         className="aspect-square object-contain w-[84px] md:w-full md:group-hover:blur-[10px] md:group-hover:brightness-[0.2] transition-all"
         src={props.imageSrc ? props.imageSrc : ''}
         alt={props.name}
       />
-      <div className="md:absolute flex md:justify-center md:items-center flex-col gap-[10px] md:gap-3 md:px-4 md:text-center md:size-full md:group-hover:bg-darkOrange-9/20 md:opacity-0 md:group-hover:opacity-100 transition-all">
-        <span className="text-white  text-sm md:text-xl md:leading-6 font-medium md:font-semibold">
+      <div className="md:absolute w-full flex md:justify-center md:items-center flex-col gap-[10px] md:gap-4 md:px-4 md:text-center md:size-full md:group-hover:bg-darkOrange-9/20 md:opacity-0 md:group-hover:opacity-100 transition-all">
+        <span className="text-white text-sm font-medium md:subtitle-large-med-20px line-clamp-3 md:line-clamp-2">
           {props.name}
         </span>
-        <span className="text-white max-md:group-hover:text-darkOrange-1 text-xs md:text-base md:leading-6 transition-all">
-          {props.author}
-          {props.year && (
-            <>
-              <span> · </span>
-              <span className="text-white/75 md:text-white max-md:group-hover:text-darkOrange-1 font-light transition-all">
-                {props.year}
+        <div className="flex justify-between flex-row items-center">
```

Comment:
Add a minimal gap (gap-1 for example) to prevent text and flag being too close
![image](https://github.com/user-attachments/assets/c123c35a-64d1-46b2-ade7-981674ad19f4)